lib/kafka_ex/protocol/offset.ex

```diff
@@ -7,6 +7,21 @@ defmodule KafkaEx.Protocol.Offset do
   defmodule Response do
     defstruct topic: "", partition_offsets: []
     @type t :: %Response{topic: binary, partition_offsets: list}
+
+    @doc """
+    Returns the numeric offset of the first partition in the response,
+    or nil if the topic is not found.
+
+    This is a convenience for the most typical case.
+    """
+    @spec first_partition_offset(atom | t) :: number | nil
+    def first_partition_offset(:topic_not_found) do
+      nil
+    end
+    def first_partition_offset([%Response{partition_offsets: partition_offsets}]) do
+      first_partition = hd(partition_offsets)
+      first_partition.offset |> hd
+    end
```

Comment:
I think this is debatably useful externally.  I pulled this in from test code I had already written in another application where I'm using KafkaEx.

This gets at a bigger question, though - why do all of these structs support arrays when the KafkaEx API doesn't really support making a request for multiple offsets of multiple topics/partitions?